examples/node/lib/rest/api/v2010/account/call.ts

```diff
@@ -301,10 +303,19 @@ export class CallInstance {
   }
 }
 
+export interface CallSolution {
+  accountSid?: string;
+}
+
 export interface CallListInstance {
+  _version: V2010;
+  _solution: CallSolution;
+  _uri: string;
+
   (testInteger: number): CallContext;
   get(testInteger: number): CallContext;
 
+  _feedbackCallSummary?: FeedbackCallSummaryListInstance;
```

Comment:
is this the same as next line? `feedbackCallSummary: FeedbackCallSummaryListInstance;`

Reply:
Should this line be removed to prevent two variables of the same name? :https://github.com/twilio/twilio-oai-generator/pull/382/files#diff-e898602ff9f10364a24da3d4c8e8be429e531a3f445d097ece1f676e95ed0102L125

Reply:
One's a `get` function and the other is caching the value of the `get` function.

Reply:
Just to clarify, so `feedbackCallSummary` is a get function that is being defined [here](https://github.com/twilio/twilio-oai-generator/pull/382/files#diff-46e160de53ef20c3902e76fde0d227f357d429e5d782453302d0561bcb2af303R361)

Reply:
Correct.